### PR TITLE
Rails 6.1 capable

### PR DESCRIPTION
**Pull request by lukebooth**

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-2.5.3
+2.5.5
```
Comment (kobsy) [inline .ruby-version]: Ruby 2.5 is technically no longer supported, I think. Do we want to bump this up to at least 2.6.x? 🤷

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+## v1.2.0 (2021 Nov 8)
+
+* Support for Rails 6.1
+* Bumped ruby patch version
```
Comment (kobsy) [inline CHANGELOG.md]: I think the `.ruby-version` might only be for development purposes and wouldn't affect a consumer of the gem. We _probably_ wouldn't need to mention it in the changelog?

**Review by kobsy — APPROVED** [review]
👍 Looks good. Thanks for bumping this up 🙂

**Comment by lukebooth** [comment]
Alright, addressed those issues

**Pull request merged**